packages/p2p/src/stores/order-store.js

```diff
@@ -64,8 +71,46 @@ export default class OrderStore {
             p2p_order_confirm: 1,
             id,
         }).then(response => {
-            if (response && response.error) {
-                order_details_store.setErrorMessage(response.error.message);
+            if (response) {
+                if (response.error) {
+                    if (response.error.code === 'OrderEmailVerificationRequired') {
```

Comment:
can we destructure`response.error`?

Reply:
I've used a ternary instead since all the conditions are inside `if (response.error) {` condition